README.md

```diff
@@ -80,4 +89,21 @@ Each lab will contain a link to the next one. Start from **"Lab 1"** and move th
 - 👩‍💻&nbsp;&nbsp;[Lab 10 - Generate Storybook stories for the shared ui component](docs/lab10/LAB.md)
 - ⌨️&nbsp;&nbsp;[Lab 11 - E2E test the shared component](docs/lab11/LAB.md)
 
+#### Day 2
+
+- 💡 [Lab 12 - Module boundaries](docs/lab12/LAB.md)
+- 🧸️ [Lab 13 - Workspace Generators - Intro](docs/lab13/LAB.md)
+- 🧵 [Lab 14 - Workspace Generators - Modifying files](docs/lab14/LAB.md)
+- 💎 [Lab 15 - Setting up CI](docs/lab15/LAB.md)
+- 🔌 [Lab 16 - Distributed caching](docs/lab16/LAB.md)
+- 🔍 [Lab 17 - NxCloud GitHub bot](docs/lab17/LAB.md)
+- 📎 [Lab 18 - Run-Commands and deploying the frontend](docs/lab18/LAB.md)
+
+| Option 1 - React frontends and more custom generators practice | Option 2 - Heroku API deployments with Docker |
```

Comment:
I think this makes sense only for Angular workshop. IMHO CI/CD track should be the only one for React

Reply:
I changed it to be a nextjs app - so they're getting exposed to deploying other kinds of frontends